src/main.rs

```diff
@@ -185,10 +191,11 @@ fn main() {
             .with_prompt("Select projects to clean")
             .report(false)
             .defaults(&preselected_projects)
-            .interact_opt() else {
-                println!("Nothing selected");
-                return;
-            };
+            .interact_opt()
+        else {
+            println!("Nothing selected");
+            return;
+        };
```

Comment:
This is a rustfmt artefact, nothing changed

Reply:
Looks like they finally added support for `let-else` to rustfmt 👍